components/suggest/src/db.rs

```diff
@@ -365,23 +374,19 @@ impl<'a> SuggestDao<'a> {
         let suggestions = self
             .conn
             .query_rows_and_then_cached(
-                r#"
```

Comment:
I do like this roomier formatting for the SQL strings that @ncloudioj used for MDN—it would be nice to adopt it elsewhere, but not at all a blocker for your PR 😄

Reply:
Ah, thanks, Lina! I stole this [script](https://mozilla.github.io/bigquery-etl/cookbooks/common_workflows/#formatting-sql) from our [pipeline project](https://github.com/mozilla/bigquery-etl) so that I don't need to manually format SQL any more. The only downside is that it could get too sparse sometimes.

Reply:
oh this is fancy